VMEncryption/test/test_encryption_settings_util.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import mock
+from main import EncryptionSettingsUtil
+from main import Common
+from StringIO import StringIO
+import console_logger
+
+class TestEncryptionSettingsUtil(unittest.TestCase):
+    """ unit tests for functions in the check_util module """
+    def setUp(self):
+        self.logger = console_logger.ConsoleLogger()
+        self.es_util = EncryptionSettingsUtil.EncryptionSettingsUtil(self.logger)
+
+    @mock.patch('time.sleep') # To speed up this test.
+    @mock.patch('main.EncryptionSettingsUtil.EncryptionSettingsUtil.write_settings_file')
+    @mock.patch('main.EncryptionSettingsUtil.EncryptionSettingsUtil.get_index')
+    @mock.patch('os.path.isfile', return_value=True)
+    @mock.patch('main.EncryptionSettingsUtil.EncryptionSettingsUtil.get_http_util')
+    def test_post_to_wire_server(self, get_http_util, os_path_isfile, get_index, write_settings_file, time_sleep):
```

Comment:
Tests of an HTTP 200, HTTP 400, and HTTP 500 response from wire server (in addition to no response) would also be helpful.